src/client/pages/store-management/components/all-invoices/all-invoices-view.html

```diff
@@ -0,0 +1,32 @@
+<div class="all-invoices-view">
+    <div class="card">
+        <header class="card-header">
+            <h2 class="card-header-title">All Invoices</h2>
+        </header>
+
+        <div class="card-content">
+            <div class="empty-invoice" v-if="invoices.length === 0">
```

Comment:
you can check whether the array is empty or not as well

Reply:
Okay